apps/acrs/behaviours/declaration.js

```diff
@@ -0,0 +1,48 @@
+const Utilities = require('../../../lib/utilities');
+
+const CheckUnder18InAggregatedValue = (aggregateArray, fieldLabel) => {
+  const aggregatedLength = aggregateArray.length;
+
+  let responseObject = '';
+  for (let i = 0; i < aggregatedLength; i++) {
+    responseObject = aggregateArray[i]
+      .fields
+      .map(v => v)
+      .filter(field => field.field === fieldLabel);
+
+    if (!Utilities.isOver18(responseObject[0].value)) {
+      return true;
+    }
+  }
+  return false;
+};
+module.exports = superclass => class extends superclass {
+  locals(req, res) {
+    const locals = super.locals(req, res);
+
+    const familyMember = req.sessionModel.get('family-member-in-uk').aggregatedValues;
```

Comment:
Again I think we may need to check with UCD but I don't think the family members in UK need to be considered when making this conditional